backend/fms_core/template_importer/importers/sample_selection_qpcr.py

```diff
@@ -53,12 +54,6 @@ def import_template_inner(self):
                 'container': {'barcode': row_data['Sample Container Barcode']},
                 'depleted': row_data['Source Depleted'],
             }
-            sample_information = {
-                'qpcr_type': row_data['qPCR Type'],
-                'ct_value_experimental': float_to_decimal_and_none(row_data['CT Value (Experimental)']),
```

Comment:
It may be a good idea to process the numeric values in the input (ct values) in order to normalize them. If we don't do that we may store number with 8 decimals positions or numbers without 0 before the decimal.